schema-lending.graphql

```diff
@@ -514,34 +755,19 @@ type Market @entity {
   " Is this market active or is it frozen "
   isActive: Boolean!
 
-  " Can you use the output token as collateral "
-  canUseAsCollateral: Boolean!
-
   " Can you borrow from this market "
   canBorrowFrom: Boolean!
 
-  " Maximum loan-to-value ratio as a percentage value (e.g. 75% for DAI in Aave) "
-  maximumLTV: BigDecimal!
-
-  " Liquidation threshold as a percentage value (e.g. 80% for DAI in Aave). When it is reached, the position is defined as undercollateralised and could be liquidated "
-  liquidationThreshold: BigDecimal!
-
-  " Liquidation penalty (or the liquidation bonus for liquidators) as a percentage value. It is the penalty/bonus price on the collateral when liquidators purchase it as part of the liquidation of a loan that has passed the liquidation threshold "
-  liquidationPenalty: BigDecimal!
-
-  " Token that need to be deposited to take a position in protocol "
-  inputToken: Token!
-
-  " Token that is minted to track ownership of position in protocol (e.g. aToken, cToken). Leave as null if doesn't exist "
-  outputToken: Token
-
   " Aditional tokens that are given as reward for position in a protocol, usually in liquidity mining programs. e.g. SUSHI in the Onsen program, MATIC for Aave Polygon "
   rewardTokens: [RewardToken!]
 
+  " All oracles that have been used for this market "
+  oracles: [Oracle!]! @derivedFrom(field: "market")
+
   ##### Quantitative Data #####
 
-  " All interest rates / fees allowed in the market. Interest rate should be in APY percentage "
-  rates: [InterestRate!]!
+  " All fees allowed in the market. Fee should be in percentage format. e.g. 0.30% liquidation fee "
+  fees: [InterestRate!]
```

Comment:
For lending protocols, rates are the most important factor, so maybe keep them in its own field?

Reply:
@tnkrxyz I moved the rates field to `TokenData`. This was to separate rates from fees. Do you mean they should all be together again?

Reply:
I can see making a second entity for fees themselves, thoughts?

Reply:
Sorry - didn't catch that (fees are separated from rates), that makes sense then, but does it make sense to make it of type "InterestRate"? Using the liquidation fee example, looks like almost all interest rate id info are irrelevant: " { Interest rate side }-{ Interest rate type }-{ Market ID }-{ Optional: # days/hours since epoch time } "

Reply:
I agree, I will update this

Reply:
@tnkrxyz can you think of any fee types aside from liquidation, protocol, and admin?

Reply:
withdraw, flashloan?